codeHF/config_tasks.sh

```diff
@@ -141,6 +149,12 @@ function AdjustJson {
 # Generate the O2 script containing the full workflow specification.
 function MakeScriptO2 {
   WORKFLOWS=""
+  # Cascade reconstruction
+  CASCADESUFFIX=""
+  [ $DOO2_CASC -eq 1 ] && CASCADESUFFIX+="_v0"
+  # Event selection
+  EVSELSUFFIX=""
+  [ $DOO2_EVSEL -eq 1 ] && WORKFLOWS+=" o2-analysis-timestamp o2-analysis-event-selection" && EVSELSUFFIX+="-evsel"
```

Comment:
* Better to use ordered names, matching the switches, like `SUFFIX_CASC`, `SUFFIX_EVSEL`.
* You can do a single line: `[ $DOO2_CASC -eq 1 ] && CASCADESUFFIX="_v0" || CASCADESUFFIX=""`.
```bash
[ $DOO2_EVSEL -eq 1 ] && WORKFLOWS+=" o2-analysis-timestamp o2-analysis-event-selection" && EVSELSUFFIX+="-evsel"
```

is not the right syntax. Instead use:

```bash
[ $DOO2_EVSEL -eq 1 ] && { WORKFLOWS+=" o2-analysis-timestamp o2-analysis-event-selection"; EVSELSUFFIX="-evsel"; }
```
* For consistency, I would replace `_v0` with `-v0` everywhere.

Reply:
Done!